config/locales/en.yml

```diff
@@ -1302,22 +1265,17 @@ en:
       button_text: "Pay now!"
       hints:
         donation: "Optional donation that will go only to the organization team"
-        payment_button: "Your payment information never reaches the WCA website. It is securely sent to Stripe, which handles the payment process and gets back to us."
       labels:
-        entry_fees: "Registration fees"
         fees_paid: "Paid"
         fees_remaining: "Remaining"
         donation: "Donation"
         show_donation: "I would like to make an additional optional donation that will go only to the organization team"
         payment_information: "Payment information"
-        payment_button: "Pay your fees with Stripe"
         ajax_error: "WCA server error"
         payment_service_error: "Payment service error"
         subtotal: "Subtotal"
       alerts:
-        not_a_number: "Please enter a valid amount."
         amount_too_low: "The total amount must be at least the registration fees to be paid."
-        amount_rather_high: "You are about to pay at least double the required amount for your registration to proceed. Please confirm the payment by pressing OK."
```

Comment:
I feel like we should include this alert. At least "steal" the string here and wrap it in a "V3 TODO"

Reply:
But then again, temporarily removing it will prompt the translators to update their translation files, and quick translators like pt and th will remove it, and later they will get an email asking them to add it back again

Reply:
yes we can add that to the PaymentStep

Reply:
done